docs/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+#### Orphan
+
+A valid block which is not built upon and therefore not included in the definitive blockchain.
+
+#### Politeia
+
+A system for facilitating the submission and discussion of [proposals](https://proposals.decred.org/) in an environment with transparent censorship. 
+
+#### Private key
+
+A string of data that allows you to access and spend DCR stored in address associated with the private key. When sending DCR from one address to another, the wallet must use the private key of the sending address to sign the transaction.
+
+#### Private passphrase
+
+A passphrate that is used to encrypt wallet.db files. wallet.db files encrypted with a private passphrase cannot be used without that passphrase.
+
+#### Proof-of-Stake (PoS) voting
+
+The mechanism by which ticket holders vote to approve blocks confirmed by PoW miners (thus providing a check on PoW miners), earn staking rewards, and vote on consensus rule changes and Politeia proposals. 
+
+#### Protocol rules
+
+See [consensus rules](#consensus-rules). 
+
+#### Public key
+
+A long string of alphanumeric characters generated algorithmically from a private key. The user’s public key is used to prove that a transaction was signed using their private key.
+
+#### Quorum
+
+The minimum level of participation required in order for a decision-making process to produce a valid outcome. Changes to the consensus rules require at least 10% of votes to be for or against the change in order to be valid. Politeaia proposals require 20%.
+
+#### Regular transaction tree
+
+The normal [transactions](https://www.reddit.com/r/decred/comments/66j4l4/decred_proof_of_stake_explained/dgjsyxd) in a block: sending DCR to an address, coinbase transactions such as PoW Miner reward and Treasury stipend.
+
+#### Reorg
+A reorganization of the blockchain in which a set of blocks are replaced by another (longer) set. The number of blocks replaced is the depth of the reorg. 
+
+#### Revoked ticket
+
+When a ticket (that was missed or expired) is revoked, the DCR used to buy it becomes spendable again.
+
+#### Rule Change Interval (RCI)
+
+An interval of 8064 blocks (~4 weeks) in which ticket holders can vote on consensus rule changes. A Rule Change Interval (RCI) consists of four Stake Version Intervals (SVI), which are 2016 blocks (~1 week). Once the conditions for a vote have been met during an SVI, voting is scheduled to begin on the first block of the next RCI. Because there are four SVIs per RCI, it can take up to 6048 blocks [3 SVIs] for the next RCI to begin.
+
+#### Rule change proposal
+
+A proposal to change the consensus rules of the Decred blockchain. Rule change proposals must be implemented in latent code within the software running the network's nodes. If the proposal passes, the latent code activates one month later. 
+
+#### Seed
+
+A series of 33 words that is used to generate the private key for a wallet. Anyone who has the seed for a wallet can spend any DCR it holds.
```

Comment:
The seed is 32 bytes of data.  The 33 word encoding is just that, an encoding.

The wallet manages many private keys, not just one.

Reply:
Rewording:

#### Seed

A 256-bit (32-byte) cryptographically secure random number that can be used to recover a wallet. When creating a wallet using any Decred wallet software, a 33 word "seed phrase" is also created from the 32-byte seed. If the wallet is lost or corrupted, the seed phrase can be used to restore the wallet's private keys, transaction history, and ballances using any Decred wallet on any computer.

Reply:
Fixed